shared/cfg/machines.cfg

```diff
@@ -46,3 +46,15 @@ variants:
         # Currently no USB support
         usbs =
         usb_devices =
+    - s390:
+        only s390
```

Comment:
Those two should have different value, otherwise avocado matches all machine types. I'd suggest calling the architecture `s390x` and machine type `s390-mmio` (or `s390-virtio` or `s390-ccw`, you know the best what reflects the machine type the most).